python/testEditorInterface.py

```diff
@@ -33,7 +33,7 @@ def main():
     # We get here when the user types a "." or ":"
 
     # argv has the name of the script as arg 1 and then user args
-    if len(sys.argv) == 4:
+    if ((len(sys.argv) == 4) or (len(sys.argv) == 5)):
```

Comment:
@aytey 
These are very quick comments, as it is the end of the day and I don't have time to look at this thoroughly

This is not good, the check for 4 was to validate that the number of args was correct for the call, just slapping 5 here makes no sense, and also invalidates the else below which says 3 args are expected.